src/modules/src/esp_deck_flasher.c

```diff
@@ -99,7 +99,7 @@ bool espDeckFlasherWrite(const uint32_t memAddr, const uint8_t writeLen, const u
   // send buffer if full
   const bool sendBufferFull = (sendBufferIndex == ESP_MTU);
   const bool lastPacket = (sequenceNumber == numberOfDataPackets - 1);
-  bool lastPacketFull;
+  bool lastPacketFull = false;
```

Comment:
nit: this _could_ be:

```c
bool lastPacketFull = lastPacket && (sendBufferIndex == ESP_BITSTREAM_SIZE % ESP_MTU);
```

And we could remove the if alltogether.... not sure it is better though

Reply:
thanks! done. I kept the if statement since I don't know if it is necessary.